src/extension.ts

```diff
@@ -357,7 +357,7 @@ export class Extension implements RunHooks {
 
     if (request.continuous) {
       for (const model of this._models.enabledModels())
-        model.addToWatch(request.include, cancellationToken!);
+        await model.addToWatch(request.include, cancellationToken!);
```

Comment:
I don't see a reason to not do that. Looking at the TestServer code it means we wait now for the Watcher to ack which seems reasonable.